### PR TITLE
Preview software overview

### PR DESCRIPTION
**Pull request by mollymccarthy03**

### CODE REVIEW

**Comment by mollymccarthy03** [comment]
https://chtc.github.io/web-preview/preview-molly-software-overview

File: _uw-research-computing/software-overview-htc.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+{% endcapture %}
+{% include /components/directory.html title="Table of Contents" %}
+
+## Quickstart
+
+Click the link in the table below to jump to the instructions for the language/program/software that you want to use.
+More information is provided in the [CHTC Recipes Repository](#chtc-recipes-repository) and [Containers](#containers) sections.
+
+<div class="card-body">
+    <button class="btn btn-guide mb-lg-0 active" onclick="showExcerpt('running-python-jobs', 'side-divider')"><img src="/uw-research-computing/guide-icons/python-icon.png" alt="pythonlogo"> Python</button>
```
Comment (CannonLock) [inline _uw-research-computing/software-overview-htc.md:73]: @mollymccarthy03 Can you make this generated based on the link like you did below?

**Review by CannonLock — CHANGES_REQUESTED** [review]
Looks good, just one tweak to make sure that any new guides are automatically added.

File: _uw-research-computing/software-overview-htc.md (hunk 1 above)
Comment (CannonLock) [reply _uw-research-computing/software-overview-htc.md:73]: I note that you need that icon url. I would suggest that rather then checking for `link == 'software'` you populate a `software_icon` with the url and check that the software_icon value is filled in instead. That way you can kill two birds with one key-value pair.

**Pull request merged**